<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="fileRoadSelect"/>
    </item>
    <item>
     <widget class="QComboBox" name="videoTypeBox">
      <item>
       <property name="text">
        <string>all</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.mp4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.mpeg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.avi</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="imageSelectBtn">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件分辨率</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="outfileRoadText"/>
    </item>
    <item>
     <widget class="QPushButton" name="outfileRoadBtn">
      <property name="text">
       <string>选择输出目录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="new_videoSizeText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_videoSizeText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前文件分辨率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>码率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_videoBitRate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>帧率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_videoFps">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>码率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_videoBitRate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>帧率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_videoFps">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="compressBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>压缩图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
